containers-int-strict/src/Map/Int.hs

```diff
@@ -2,7 +2,10 @@
 {-# LANGUAGE DerivingStrategies         #-}
 {-# LANGUAGE FlexibleInstances          #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE RankNTypes                 #-}
```

Comment:
I think you don't need `RankNTypes` extension

`ScopedTypeVariables` implies `ExplicitForAll` extension, and you don't have rank-2 types in this module (yet).